sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/filesystem/InputComponentStore.java

```diff
@@ -129,7 +126,7 @@ public InputComponentStore put(InputFile inputFile) {
     inputFileCache.put(file.moduleKey(), inputFile.relativePath(), inputFile);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "relativePath"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)